.github/workflows/test_conda_store_server_integration.yaml

```diff
@@ -38,18 +39,14 @@ jobs:
       - name: "Checkout Repository 🛎"
         uses: actions/checkout@v4
 
-      - name: "Get project's default Python version 🏷️"
-        run: |
-          echo "PYTHON_VERSION_DEFAULT=$(cat .python-version-default)" >> $GITHUB_ENV
-
       - name: "Set up conda env 🐍"
         uses: conda-incubator/setup-miniconda@v3
         with:
             environment-file: conda-store-server/environment-dev.yaml
             miniforge-version: latest
             auto-activate-base: false
             activate-environment: conda-store-server-dev
-            python-version: ${{ env.PYTHON_VERSION_DEFAULT }}
+            python-version: ${{ matrix.python-version }}
```

Comment:
Same note as above about needing to also update the Python runtime version in the container and not only in the GH workflow

Reply:
I'm actually going to keep this change because we do run `pytest` with this environment, and the unit tests will actually use this python version.